scio-parquet/src/main/scala/com/spotify/scio/parquet/types/ParquetTypeIO.scala

```diff
@@ -112,12 +96,10 @@ object ParquetTypeIO {
   object ReadParam {
     private[types] val DefaultPredicate = null
     private[types] val DefaultConfiguration = new Configuration()
-    private[types] val DefaultSkipClone = true
   }
   final case class ReadParam[T] private (
     predicate: FilterPredicate = null,
-    conf: Configuration = ReadParam.DefaultConfiguration,
-    skipClone: Boolean = ReadParam.DefaultSkipClone
+    conf: Configuration = ReadParam.DefaultConfiguration
```

Comment:
cc @lofifnc - the `skipClone` param is removed in the new splittable DoFn implementation of Parquet, which doesn't rely on `HadoopFormatIO`.

Reply:
This is great! Sayonara `HadoopFormatIO`!